lib/request-helpers.js

```diff
@@ -84,3 +85,72 @@ const setCacheControlForStatsResponse = (res, filter) => {
     res.setHeader('cache-control', `public, max-age=${365 * 24 * 3600}, immutable`)
   }
 }
+
+/**
+ * @param {object} args
+ * @param {pg.Pool} args.pgPool
+ * @param {string} args.table
+ * @param {string} args.column
+ * @param {import('./typings').Filter} args.filter
+ * @param {string} [args.asColumn]
+ */
+export const getDailyDistinctCountQuery = async ({
```

Comment:
Since this function executes the query instead of returning the query, I think it should not have `Query` in the name.

```suggestion
export const getDailyDistinctCount = async ({
```